engine/install/debian.md

```diff
@@ -37,7 +37,7 @@ Uninstall any such older versions before attempting to install
 a new version:
 
 ```console
-$ sudo apt-get remove docker docker-engine docker.io containerd runc
+$ for pkg in docker docker-engine docker.io containerd runc; do sudo apt-get remove -y $pkg; done
```

Comment:
Not sure about `-y`. I think it's better to let the user confirm before deleting anything, even if that means he'll have to do it 5 times.
User can always add `-y` by himself if he knows what he's doing.